src/gleam/string.gleam

```diff
@@ -309,15 +309,17 @@ pub fn join(strings: List(String), with separator: String) -> String {
 //
 //
 // pub fn pad_right(string: String, to size: Int, with: String) {}
-// TODO
+
 // Get rid of whitespace on both sides of a String.
 //
 // ## Examples
 //    > trim("  hats  \n")
 //    "hats"
 //
 //
-// pub fn trim(string: String) -> String {}
```

Comment:
Let's use doc comments for these `///` so that this is included in the HTML docs